saw-script/src/SAWScript/Interpreter.hs

```diff
@@ -3638,137 +4072,160 @@ primitives = Map.fromList $
     (pureVal goal_has_some_tag)
     Experimental
     [ "Returns true if the current goal is tagged with any the tags"
     , "in the given list. This function returns false for all goals"
     , "when given an empty list. Tags may be added to goals using"
-    , "'llvm_setup_with_tag' and similar operations in the specification"
-    , "setup phase."
+    , "'llvm_setup_with_tag' and similar operations in the"
+    , "specification setup phase."
     ]
-  , prim "goal_num_ite"       "{a} Int -> ProofScript a -> ProofScript a -> ProofScript a"
-    (pureVal goal_num_ite)
+
+    ------------------------------------------------------------
+    -- Focus-related proof tactics
+
+  , prim "unfocus"        "ProofScript ()"
+    (pureVal unfocus)
     Experimental
-    [ "If the goal number is the given number, runs the first script."
-    , "Otherwise runs the second script" ]
-  , prim "goal_num_when"       "Int -> ProofScript () -> ProofScript ()"
-    (pureVal goal_num_when)
+    [ "Remove any sequent focus point." ]
+
+  , prim "focus_concl"      "Int -> ProofScript ()"
+    (pureVal focus_concl)
     Experimental
-    [ "Run the given proof script only when the goal number is the"
-    , "the given number."
-    ]
-  , prim "print_goal"          "ProofScript ()"
-    (pureVal print_goal)
-    Current
-    [ "Print the current goal that a proof script is attempting to prove." ]
-  , prim "print_goal_inline"   "[Int] -> ProofScript ()"
-    (pureVal print_goal_inline)
-    Current
-    [ "Print the current goal that a proof script is attempting to prove,"
-    , "without generating `let` bindings for the provided indices. For"
-    , "example, `print_goal_inline [1,9,3]` will print the goal without"
-    , "inlining the variables that would otherwise be abstracted as `x@1`,"
-    , " `x@9`, and `x@3`. These indices are assigned deterministically with"
-    , "regard to a particular goal, but are not persistent across goals. As"
-    , "such, this should be used primarily when debugging a proof."
-    , ""
-    , "Note: incompatible with non-incremental memoization strategies - see"
-    , "`set_memoization_incremental` and `set_memoization_hash_incremental`."
-    ]
-  , prim "write_goal" "String -> ProofScript ()"
-    (pureVal do_write_goal)
-    Current
-    [ "Write the current goal that a proof script is attempting to prove"
-    , "into the named file."
+    [ "Focus on the numbered conclusion within a sequent. This will"
+    , "fail if there are not enough conclusions."
     ]
-  , prim "print_goal_summary" "ProofScript ()"
-    (pureVal print_goal_summary)
-    Current
-    [ "Print the number and description of the goal that a proof script"
-    , "is attempting to prove."
+
+  , prim "focus_hyp"       "Int -> ProofScript ()"
+    (pureVal focus_hyp)
+    Experimental
+    -- XXX this description doesn't make sense
+    [ "Focus on the numbered conclusion with a sequent.  This will fail"
+    , "if there are enough hypotheses."
```

Comment:
I think that the use of "Focus on the numbered conclusion" in this help text is an accidental copy-paste mistake introduced by copying the help text from `focus_concl`. I would suggest changing "numbered conclusion" to "numbered hypothesis".